uiowa/uiowa
# Cleanup after getting HR off the island

* Resolves #1907 
* Resolves #1713 
* Resolves #3719 
* ~Changes location of `uiowa_*` modules as outlined in #2629, but does not move `uiowa_auth` into this project.~ - Holding off on this until we can get `uiowa_auth` moved into the main project.
* Resolves #2629

# How to test

* Code review of changed files and skimming the deletions to make sure they make sense.
* `composer install`
* `blt ds --site hr.uiowa.edu`
* Ensure sync happens without error.
* `drush @hr.local uli`
* Spot check HR site locally to make sure there are no issues.
  * You can save card paragraphs without error
  * Article full images are displaying properly
  * Footer contact info is still present.
  * You can hide/show titles/sidebar on page content
